Editor/Modules/TextureAnalyzer.cs

```diff
@@ -97,6 +98,21 @@ public class TextureAnalyzer : ITextureModuleAnalyzer
             }
         };
 
+        internal static readonly Descriptor k_TextureSolidColorDescriptor = new Descriptor(
+            PAT0004,
+            "Texture: Solid color is not 1x1 size",
+            new[] {Area.Memory},
+            "The texture is a solid color. This increases the amount of memory usage and can be reduced.",
+            "Consider shrinking the texture to 1x1 format."
```

Comment:
Oh, just a last wording change.

It is maybe clearer to write "size" instead of "format", like in the other sentences.

I think the PR is good then!